### PR TITLE
Add prev/next change buttons in Quick Action Bar

### PR DESCRIPTION
**Pull request by anilsenay**
Release Notes:

- Added buttons for jumping to previous/next changes (aka hunks)

In vscode, there are arrow buttons for jumping to changed lines in Working Tree mode.
resim

Similarly, I added prev/next buttons for same functionality to Quick Action Bar _if file has git diff_.

I was unaware of hunk commands until lately, thus I believe it is more user-friendly to demonstrate that Zed provides this functionality.
Could you please share your thoughts? I'd appreciate it.

https://github.com/zed-industries/zed/assets/1047345/bb6e7ac3-0c19-4b6c-b15c-076292bad9d9

### CODE REVIEW

File: crates/quick_action_bar/src/quick_action_bar.rs [hunk 1]
```diff
@@ -86,6 +87,13 @@ impl Render for QuickActionBar {
             return div().id("empty quick action bar");
         };
 
+        let has_git_diffs = editor
+            .read(cx)
+            .buffer()
+            .read(cx)
+            .snapshot(cx)
+            .has_git_diffs();
```
Comment (anilsenay) [inline crates/quick_action_bar/src/quick_action_bar.rs:95]: is there any better way for this?

Comment (anilsenay) [inline crates/quick_action_bar/src/quick_action_bar.rs:95]: is there any better way for this?

**Comment by iamnbutler** [comment]
We don't currently have UI buttons for other movement commands in the tool bar, and adding icons here isn't something we want to do for most things. Keeping the UI minimal is important to us. If we were to consider something like this, it should be part of a system that allows users to customize the toolbar more widely (think finder on macOS:)

CleanShot 2024-05-21 at 16 55 14@2x

We do have some upcoming UI that this would slot into better: 
CleanShot 2024-05-21 at 16 58 38@2x

I could imagine next/previous hunk as a section between the top set of buttons and the bottom set here.

**Comment by anilsenay** [comment]
Oh, got it. Mine was just a suggestion. Upcoming UI looking so nice 🚀

**Pull request closed without merging**